insights/specs.py

```diff
@@ -355,6 +355,7 @@ class Specs(SpecSet):
     udev_persistent_net_rules = RegistryPoint(alias="udev-persistent-net.rules")
     uname = RegistryPoint()
     up2date = RegistryPoint()
+    up2date_log = RegistryPoint(alias="up2date_log")
```

Comment:
`alias` is used to keep compatible with the name in 1.x which contains invalid characters (`.` or `-`), For the newly added specs, it's good to use the same name in both 1.x and 3.x